docs/source/install.rst

```diff
@@ -164,4 +164,10 @@ To deactivate the ``conda`` environment:
 
 .. code-block:: bash
 
-    (env_zipline) $ source deactivate
+    (env_zipline) $ conda deactivate
+
+.. note::
```

Comment:
Actually I'm not entirely sure whether we want to use the `note` directive or just a normal text here.